monai/apps/deepgrow/dataset.py

```diff
@@ -126,8 +126,8 @@ def _default_transforms(image_key, label_key, pixdim):
         [
             LoadImaged(keys=keys),
             AsChannelFirstd(keys=keys),
-            Spacingd(keys=keys, pixdim=pixdim, mode=mode),
```

Comment:
the orientation should happen first, could you please help evaluate the impact of this change? most likely we don't need to retrain any existing models... @SachidanandAlle @finalelement @diazandr3s.

Reply:
yes.. this should be ok.. we change the orientation while training the model..  for existing pre-trained models.. this should not impact..

Reply:
ok, that's right thanks for confirming it.

Reply:
Agree with Sachi. We are already using DeepEdit without the Spacing transform :)